bulkwebhook/bulk_webhook/doctype/kafka_hook/kafka_hook.py

```diff
@@ -7,9 +7,11 @@
 from frappe import _
 from frappe.model.document import Document
 from frappe.utils.jinja import validate_template
-from frappe.utils.safe_exec import get_safe_globals
 from bulkwebhook.bulk_webhook.doctype.kafka_settings.kafka_utlis import send_kafka
 from bulkwebhook.bulk_webhook.doctype.bulk_webhook.bulk_webhook import log_request
+from frappe.integrations.doctype.webhook.webhook import get_context as get_webhook_context
+
+WEBHOOK_CONTEXT = get_webhook_context(doc=None)
```

Comment:
let's just hardcode it IMO. maybe the context in webhook changes and contains values that cannot be set as a global?

also the `get_safe_globals` call access objects that need a corresponding local (e.g. `frappe.db`) - this may fail if the local isn't initialized 😅

Reply:
Yeah, adding utils directly is better 😄 - will do that!